frontend/web/styles/project/_lists.scss

```diff
@@ -59,3 +59,42 @@
     background-color: $bg-dark300;
   }
 }
+
```

Comment:
Is it possible to use existing UI and classnames for the metadata work? I can't see anything that'd require new css

Reply:
With the new changes, they are no longer necessary, removed.